awswrangler/data_quality/_create.py

```diff
@@ -0,0 +1,157 @@
+        Ruleset description.
+
+    Examples
+    --------
+    >>> import awswrangler as wr
+    >>> wr.data_quality.create_ruleset(
+    >>>     name="ruleset",
+    >>>     database="db",
+    >>>     table="table",
+    >>>     dqdl_rules=dqdl_rules,
+    >>>)
+    """
+    if df_rules is not None and dqdl_rules:
+        raise exceptions.InvalidArgumentCombination("You must pass either ruleset `df_rules` or `dqdl_rules`.")
+
+    client_glue: boto3.client = _utils.client(service_name="glue", session=boto3_session)
+    dqdl_rules = _create_dqdl(df_rules) if df_rules is not None else dqdl_rules
+
+    try:
+        client_glue.create_data_quality_ruleset(
+            Name=name,
+            Description=description,
+            Ruleset=dqdl_rules,
+            TargetTable={
+                "TableName": table,
+                "DatabaseName": database,
+            },
+            ClientToken=client_token if client_token else uuid.uuid4().hex,
+        )
+    except client_glue.exceptions.AlreadyExistsException:
+        raise exceptions.AlreadyExists(f"Ruleset {name} already exists.")
+
+
+@apply_configs
+def evaluate_ruleset(
+    name: str,
+    iam_role_arn: str,
+    number_of_workers: int = 5,
+    timeout: int = 2880,
+    database: Optional[str] = None,
+    table: Optional[str] = None,
+    catalog_id: Optional[str] = None,
+    connection: Optional[str] = None,
+    additional_run_options: Optional[Dict[str, str]] = None,
+    boto3_session: Optional[boto3.Session] = None,
+):
+    """Evaluate Data Quality ruleset.
+
+    Parameters
+    ----------
+    name : str
+        Ruleset name.
+    iam_role_arn : str
+        IAM Role.
+    number_of_workers: int, optional
+        The number of G.1X workers to be used in the run. The default is 5.
+    timeout: int, optional
+        The timeout for a run in minutes. The default is 2880 (48 hours).
+    database : str, optional
+        Glue database name. Database associated with the ruleset will be used if not provided.
+    table : str, optinal
+        Glue table name. Table associated with the ruleset will be used if not provided.
+    catalog_id : str, optional
+        Glue Catalog id.
+    connection : str, optional
+        Glue connection.
+    additional_run_options : Dict[str, str], optional
+        Additional run options.
+    boto3_session : boto3.Session, optional
+        Ruleset description.
+
+    Examples
+    --------
+    >>> import awswrangler as wr
+    >>> wr.data_quality.evaluate_ruleset(
+    >>>     name="ruleset",
+    >>>     iam_role_arn=glue_data_quality_role,
+    >>> )
+    """
+    run_id: str = _start_ruleset_evaluation_run(
+        ruleset_names=[name],
+        iam_role_arn=iam_role_arn,
+        number_of_workers=number_of_workers,
+        timeout=timeout,
+        database=database,
+        table=table,
+        catalog_id=catalog_id,
+        connection=connection,
+        additional_options=additional_run_options,
```

Comment:
I think you are mixing up `additional_options` and `additional_run_options`. When you look at the definition for [start_data_quality_ruleset_evaluation_run](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.start_data_quality_ruleset_evaluation_run), there's `additional_options` when specifying the data source, and `additional_run_options` which allows us to publish the results to CloudWatch.

Looking at this function definition, I would expect `additional_run_options` to map to `additional_run_options, not to `additional_options`. I think we should include both, and specify what each allows us to set.